src/constants/brand.constants.ts

```diff
@@ -0,0 +1,20 @@
+export const brandConfig = {
```

Comment:
It would be great if we can add a short JSDoc for this so when the consumer hovered on this can see some details of what we have here